src/main/java/de/aservo/atlassian/confapi/model/SettingsBean.java

```diff
@@ -17,18 +17,18 @@
     @XmlElement
     private String baseUrl;
 
+    @XmlElement
+    private String mode;
+
     @XmlElement
     private String title;
 
-    @Override
-    public int hashCode() {
-        return HashCodeBuilder.reflectionHashCode(this);
-    }
+    public String getMode() {
+        if (isNotBlank(mode)) {
+            return mode.toLowerCase();
+        }
```

Comment:
warum hier der Aufwand mit dem lowercase? lass doch so wie default.

Reply:
Der Mode wird in Jira als Lowercase Value gespeichert. Durch den Getter wird erreicht, dass man überall, wo der Bean verwendet wird, die Nutzereingabe als Lowercase erhält, was die Fehleranfälligkeit reduziert.

Reply:
alles klar!